packages/playwright-test/src/loader.ts

```diff
@@ -363,19 +379,21 @@ class ProjectSuiteBuilder {
         }
       } else {
         const test = entry._clone();
+        to._addTest(test);
         test.retries = this._project.retries;
-        // We rely upon relative paths being unique.
```

Comment:
This comment is still relevant. Also, to be exactly correct, this code and `getClashingTestsPerSuite` must use the same delimiter (space vs ">").